### PR TITLE
fix fabric bundle weight and volume

### PR DESCRIPTION
**Pull request by nornagon**
#### Summary
Bugfixes "Fixed textile bundle sizes to be 100 times the individual items"

#### Purpose of change

Weight and volume for fabric bundles were wildly wrong.

Ref https://github.com/CleverRaven/Cataclysm-DDA/issues/47630

#### Describe the solution

This fixes the volume and weight for fabric bundles, and also ups the size of the bundles to be a larger number of items instead of 10. The idea is that 10 items is not that hard to deal with.

- bundle_rag renamed to "bundle of cotton patches", size is now 100 instead of 10
- bundle_leather size is now 50 instead of 10
- bundle_wool size is now 100 instead of 10
- introduced bundle_sheet "bundle of cotton sheets" at a size of 25

these numbers are chosen to roughly keep the bundle size around 0.5 L - 1 L in volume.

#### Describe alternatives you've considered

Could leave the bundles at size 10. Maybe this would be better for savegame compatibility?

### CODE REVIEW

**Comment by ANickelN** [comment]
Would you add the new "bundle of cotton sheets" to the bundle_of_items group located in nested.json?

**Comment by nornagon** [comment]
brought leather bundle size down to 50, to keep bundle volume around 500 g - 1 L for all fabrics

**Comment by nornagon** [comment]
The density check here is failing, but i think it would also fail for leather?

https://github.com/CleverRaven/Cataclysm-DDA/blob/b5e0cb58afcfea8ea3f86fbba4853a12b12aad59/data/json/items/resources/tailoring.json#L161-L162

this would be a density of 1.6 g/ml for leather patch, which is above the 1.1 density for the leather material.

why is this test failing for the bundle but not for the individual patch?

**Comment by nornagon** [comment]
ah: https://github.com/CleverRaven/Cataclysm-DDA/blob/b5e0cb58afcfea8ea3f86fbba4853a12b12aad59/data/mods/TEST_DATA/known_bad_density.json#L1077

i'll add the bundle to that list then, since it's just a flat multiplier on the underlying item.

**Comment by Maleclypse** [comment]
> The density check here is failing, but i think it would also fail for leather?
> 
> https://github.com/CleverRaven/Cataclysm-DDA/blob/b5e0cb58afcfea8ea3f86fbba4853a12b12aad59/data/json/items/resources/tailoring.json#L161-L162
> 
> this would be a density of 1.6 g/ml for leather patch, which is above the 1.1 density for the leather material.
> 
> why is this test failing for the bundle but not for the individual patch?

Probably means we know that the patch is wrong but no one has fixed it yet because there is a long list of wrong densities so any item that copies from the original patch won't benefit from that wrongness.

File: data/mods/TEST_DATA/known_bad_density.json [hunk 1]
```diff
@@ -1075,6 +1075,7 @@
       "test_waterproof_bag",
       "fake_goggles",
       "leather",
+      "bundle_leather",
```
Comment (anothersimulacrum) [inline data/mods/TEST_DATA/known_bad_density.json:1078]: Please don't add to this list. We'll never get through it if we allow that.

**Comment by nornagon** [comment]
How should I resolve this, then? I could edit either the weight or the volume of the leather patch to match the density of the leather material. I could edit the density of leather. There are sources which claim the density of leather as being anything from [0.2–0.3](https://www.researchgate.net/figure/density-of-each-leather-type-at-ambient-conditions_tbl1_328549546) to [0.86](https://www.engineeringtoolbox.com/density-solids-d_1265.html) to [1.5+](https://nvlpubs.nist.gov/nistpubs/jres/31/jresv31n3p169_A1b.pdf) g/ml in some cases.

Which source are we choosing here?

I'd prefer to avoid this PR ballooning into a reevaluation of the density of all leather items.

**Comment by Maleclypse** [comment]
I would suggest this "adjust leather patch to match the density of the leather material." path

**Comment by nornagon** [comment]
Okay! Sounds reasonable. There are two options then, for a leather density of 1.1 g/ml:

1. keep weight at 37 g, adjust volume to 34 ml
2. keep volume at 23 ml, adjust weight to 25 g

does one of these options strike you as better than the others?

**Comment by Maleclypse** [comment]
I think option 2 makes the most sense as most of our tailoring recipes are balanced towards volume rather than weight

**Comment by nornagon** [comment]
alrighty, done. I also upped the volume of leather_bundle a bit to fit within the max density. The other bundles allow for a bit of compression (i.e. you're cinching the thread tight to increase the density of the fabric) but for leather I figure it's not very compressible, so I stuck to flat 50x volume, 50x weight for the leather bundle.

**Pull request merged**